### PR TITLE
Update unshorten url with slack

### PR DESCRIPTION
**Pull request by mcochran-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Unshorten_URL_with_Slack/Unshorten_URL_with_Slack.icon [hunk 1]
```diff
@@ -386,22 +521,22 @@
...
             "nodeId": "96b62e72-3067-4505-9efc-be3cce6fe8b7",
-            "name": "Print Unshortened URL to Slack",
+            "name": "Print Unshortened URL",
             "type": "action_chatops",
             "continueOnFailure": false,
             "isDisabled": false,
             "parameters": {
               "input": {
                 "attachments": [],
                 "message": ":notebook_with_decorative_cover: URL Unshorten for <http://rapid7.com|{{[6f80dc1f-9a48-4fbd-bfa0-06f2a86a6c04].[content]}}> :notebook_with_decorative_cover:\n_WhoIs_\n> Shortened URL: <http://rapid7.com|{{[6f80dc1f-9a48-4fbd-bfa0-06f2a86a6c04].[content]}}>\n> Full URL: <http://rapid7.com|{{[fe7505b0-7be8-41ee-865f-420850c69bbe].[content]}}>",
```
Comment (sengleson-r7) [inline workflows/Unshorten_URL_with_Slack/Unshorten_URL_with_Slack.icon:531]: We have the wrong variables and hyperlinks here! All these are configured to go to "http://rapid7.com" =)

**Review by sengleson-r7 — CHANGES_REQUESTED** [review]
Two change requests:
1. Replace the "http://rapid7.com" links with the actual variables from the workflow. I am not sure if it is possible, but if we can _not_ hyperlink that may be best, seeing as these may be malicious URLs in the real world!
2. Add a link to the Job in the confirmation message

File: workflows/Unshorten_URL_with_Slack/Unshorten_URL_with_Slack.icon (hunk 1 above)
Comment (mcochran-r7) [reply workflows/Unshorten_URL_with_Slack/Unshorten_URL_with_Slack.icon]: This should be fixed now

**Comment by mcochran-r7** [comment]
Changes made workflow should be ready for review

**Review by ewilson-r7 — APPROVED** [review]
Ran through a couple of tests in Anonymoose

**Review by sengleson-r7 — APPROVED** [review]

**Pull request merged**